packages/ember-glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -0,0 +1,545 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { strip } from '../../utils/abstract-test-case';
+import { set } from 'ember-metal/property_set';
+import { get } from 'ember-metal/property_get';
+import setProperties from 'ember-metal/set_properties';
+import Component from 'ember-views/components/component';
+import EmberObject from 'ember-runtime/system/object';
+import { A as emberA } from 'ember-runtime/system/native_array';
+
+moduleFor('Helpers test: {{unbound}}', class extends RenderingTest {
+
+  ['@htmlbars it should not update the value if the parent view doesn\'t rerender']() {
+    this.render(`{{unbound foo}} {{unbound bar}}`, {
+      foo: 'BORK',
+      barBinding: 'foo'
+    });
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'OOF'));
+
+    this.assertText('BORK BORK');
+
+    this.runTask(() => set(this.context, 'foo', 'BORK'));
+
+    this.assertText('BORK BORK');
+  }
+
+  ['@htmlbars should be able to output a property without binding']() {
+    this.render(`<div id="first">{{unbound content.anUnboundString}}</div>`, {
+      content: EmberObject.create({
+        anUnboundString: 'No spans here, son.'
+      })
+    });
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => set(this.context, 'content.anUnboundString', 'HEY'));
+
+    this.assertText('No spans here, son.');
+
+    this.runTask(() => set(this.context, 'content', EmberObject.create({
+      anUnboundString: 'No spans here, son.'
+    })));
+
+    this.assertText('No spans here, son.');
+  }
+
+  ['@htmlbars should be able to use unbound helper in #each helper']() {
+    this.render(`<ul>{{#each view.items as |item|}}<li>{{unbound item}}</li>{{/each}}</ul>`, {
+      items: emberA(['a', 'b', 'c', 1, 2, 3])
+    });
+
+    this.assertText('abc123');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('abc123');
+
+    this.runTask(() => set(this.context.items, 'firstObject', 'hello'));
```

Comment:
Yea so that doesn't actually work. `firstObject` is a read-only property. This however does mutate the array `Ember.set(this.context.items, '0', 'wat');`. Not sure if we should test that or just do what you mentioned above.